index.js

```diff
@@ -51,7 +56,7 @@ const makeDir = async (input, options) => {
 	const stat = promisify(options.fs.stat);
 
 	if (useNativeRecursiveOption && options.fs.mkdir === fs.mkdir) {
-		const pth = path.resolve(input);
+		const pth = getTargetDir(input);
```

Comment:
The `.resolve` call is moot anyway here I think as the native method resolves internally.